core/arch/arm/plat-imx/registers/imx6-regs.h

```diff
@@ -3,7 +3,6 @@
  * Copyright 2017-2018 NXP
  *
```

Comment:
Well, here there should be "Copyright (c) 2016, Wind River Systems." and the full license text. See their last contrib: 
https://github.com/OP-TEE/optee_os/commit/ea850f1fe8b9354a97351939656f889d0ff338a4.